Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -57,6 +57,7 @@ class XMLHttpRequestBase {
     this.onreadystatechange = null;
     this.onload = null;
     this.upload = undefined; /* Upload not supported yet */
+    this.timeout = 0;
```

Comment:
@mkonicek I don't think so. It's set externally so it needs to be public.

E.g 

`````javascript
    var request = new XMLHttpRequest();
    request.timeout = 3100;
    ...
`````